src/openrct2/entity/Peep.cpp

```diff
@@ -808,7 +808,7 @@ void Peep::UpdateFalling()
                                      { x, y }, tile_element->AsPath()->GetSlopeDirection(), tile_element->AsPath()->IsSloped())
                     + tile_element->GetBaseZ();
 
-                if (height < z - 1 || height > z + 4)
+                if (height < z - 1 || height > z + 8)
```

Comment:
Should we use a constant here?
```suggestion
                if (height < z - 1 || height > z + kCoordsZStep)
```

Reply:
I don’t think so. It just so happens to be the same value. Using the kCoordsZStep constant suggests a link that isn’t there.